icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java

```diff
@@ -304,8 +304,8 @@
         /*
          * Returns the identifier of the prefix.
          *
-         * @draft ICU 68
-         * @provisional This API might change or be removed in a future release.
+         * @internal
+         * @deprecated This API is ICU internal only.
```

Comment:
You also need `@Deprecated` annotation to this method, not just API doc `@deprecated`.
For your reference, run `ant checkAPIStatus` to see warnings/errors.

Reply:
Thanks @yumaoka 

I still get this 
```
  [javadoc] warning: no tag for com.ibm.icu.number.Precision.BogusRounder()
  [javadoc] warning: no tag for com.ibm.icu.number.Precision.BogusRounder.apply(com.ibm.icu.impl.number.DecimalQuantity)
  [javadoc] warning: no tag for com.ibm.icu.util.MeasureUnit.SIPrefix.getIdentifier()
  [javadoc] warning: no tag for com.ibm.icu.util.MeasureUnit.SIPrefix.getPower()
```

what is the reason?

Reply:
done!

but why still a warning for 
```
[javadoc] warning: no tag for com.ibm.icu.number.Precision.BogusRounder()
  [javadoc] warning: no tag for com.ibm.icu.number.Precision.BogusRounder.apply(com.ibm.icu.impl.number.DecimalQuantity)
  ```

Reply:
I noticed BogusRounder doesn't need to be public. I'll push a commit to this branch.

Reply:
My bad: "ant checkAPIStatus" doesn't try to build it... :facepalm: - reverted.

Reply:
I handled all issues except SIPrefix - https://github.com/unicode-org/icu/pull/1371
I planned to update SIPrefix based on the discussion, but you already went ahead and created PR - so I avoided merge conflict.